pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/SchemasResource.java

```diff
@@ -461,11 +465,28 @@ public void getVersionBySchema(
     }
 
     private static GetSchemaResponse convertSchemaAndMetadataToGetSchemaResponse(SchemaAndMetadata schemaAndMetadata) {
+        String schemaData;
+        if (schemaAndMetadata.schema.getType() == SchemaType.KEY_VALUE) {
```

Comment:
Can you add test cases for this code path as well?

Reply:
> Can you add test cases for this code path as well?

OK, I will add the tests